README.md

```diff
@@ -1,23 +1,145 @@
-# ttoss Monorepo Template
+<h1 align="center">
 
-Monorepo template used by [Triãngulos Tecnologia](https://triangulostecnologia.com) and ttoss projects.
+`LPS - Landing Pages`
 
-## Features
+</h1>
 
-- Eslint
-- Commitlint
-- Husky
-- Jest
-- Lint Staged
-- NPM Check Updates
-- Prettier
-- Stylelint
-- TypeScript
+<p align="center">
+<a href="https://github.com/TriangulosTecnologia/lps/blob/main/LICENSE"></a>
+    <img src="https://img.shields.io/badge/license-MIT-green" alt="License" />
+  </a>
+</p>
 
-## Init Husky
+<br/>
+
+## 📚 About
+
+<strong> Landing Page</strong> is a page that has all the elements aimed at converting the visitor into Lead, opportunity or customer. Also as landing pages or conversion pages, Landing Pages are widely used in Digital Marketing campaigns, as they tend to have high conversion rates.
+
+## 🛠 Design pattern
+
+<p align="justify">
+<a href="https://github.com/ttoss/monorepo"><strong> Monorepo </strong></a> (ttoss Monorepo Template) as the name says, would be a single repository to keep the code of several projects. Each app domain is within this repository: backend, IoT, machine learning, web and mobile apps. This repository structure is useful because we can access any part of the application without changing the repository.
+</p>
+
+<br/>
+
+## 🚀 Projects
+
+- **[Simulare](https://github.com/TriangulosTecnologia/lps/tree/main/lps/simulare)**
+
+<br/>
+
+## 📌 Technologies
+
+- [TypeScript](https://www.typescriptlang.org/)
+- [Next.js](https://nextjs.org/)
+- [Fullpage.js](https://alvarotrigo.com/react-fullpage/)
+- [ThemeUI](https://theme-ui.com/)
+- [React Hook Form](https://react-hook-form.com/)
+- [Yup](https://github.com/jquense/yup)
+- [React.js](https://reactjs.org/)
+- [ESlint](https://eslint.org/)
+- [Husky](https://typicode.github.io/husky/#/)
+- [Commitlint](https://commitlint.js.org/#/)
+- [Jest](https://jestjs.io/)
+- [Lint Staged](https://github.com/okonet/lint-staged#readme)
+- [NPM Check Updates](https://github.com/raineorshine/npm-check-updates)
+- [Prettier](https://prettier.io/)
+- [Stylelint](https://stylelint.io/)
+
+<br/>
+
+## 🔖 Design
+
+We've adopted some specifications to define our <a href="https://theme-ui.com/theming/"><strong> Theme UI theme </strong></a>. You can check our <a href="https://github.com/TriangulosTecnologia/lps/blob/main/lps/simulare/src/theme.ts"><strong> full theme specifications here</strong></a>.
+
+</p>
+
+- ## Breakpoints
+
+We defined six breakpoints `['40rem', '48rem', '64rem', '80rem', '96rem', '120rem']`, which is equivalent to screens with `[640, 768, 1024, 1280, 1536, 1920]` pixels, if the root font size is equal to `16px`. Defining this array of breakpoints comes in handy because we can work with <a href="https://styled-system.com/guides/array-props/"><strong>array props </strong></a> when we need to implement responsiveness.
```

Comment:
We defined six breakpoints ['40rem', '48rem', '64rem', '80rem', '96rem', '120rem'], which is equivalent to screens with [640, 768, 1024, 1280, 1536, 1920] pixels, if the root font size is equal to 16px. Defining this array of breakpoints comes in handy because we can work with array props to implement responsiveness.